File: baekjoon/java/10807. 개수 세기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int count = 0;
        int[] arr = new int[n];
        for (int i=0;i<n;i++){
            arr[i] = sc.nextInt();
        }
        int question = sc.nextInt();
        for (int a : arr){
            if (a == question) count++;
        }
        System.out.println(count);
    }
}

File: baekjoon/cpp/10807. 개수 세기.cpp
#include <iostream>
using namespace std;


int main(){
    int n, question, cnt = 0;
    cin >> n;
    int arr[n];

    for (int i = 0; i < n;i++){
        cin >> arr[i];
    }
    cin >> question;
    for (int a : arr){
        if (a == question)
            cnt++;
    }
    cout << cnt;
    return 0;
}
